src/app/views/collections/validation/collectionValidation.js

```diff
@@ -27,7 +27,7 @@ export default class collectionValidation {
         }
 
         let nameTrimmed = name.trim();
-        let nameReplaced = name.replace(/\s+/g, "");
+        let nameReplaced = name.replace(/\s+/g, "").toLowerCase();
```

Comment:
Any chance we could have test coverage for this please?

Maybe also worth renaming the variable?